Makefile

```diff
@@ -33,6 +35,8 @@ IMAGE_NAME ?= ramen
 IMAGE_TAG ?= latest
 IMAGE_TAG_BASE = $(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)
 RBAC_PROXY_IMG ?= "gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0"
+OPERATOR_SUGGESTED_NAMESPACE ?= ramen-system
+AUTO_CONFIGURE_DR_CLUSTER ?= true
```

Comment:
Looking at e2e scripts to understand what changes are required, but opening for review anyway.